R/occDetFunc.r

```diff
@@ -51,6 +51,7 @@
 #' Defaults to 1, in which case the model is applied for so long there is a single record of the focal species.
 #' @param provenance An optional text string allowing the user to identify the dataset.
 #' @param rem_aggs_with_missing_regions An option which if TRUE will remove all aggregates which contain at least one region with no data.
+#' @param allowSitesMultiRegions An option that permits sites to be included in more than one region. If `FALSE` then these sites are dropped.
 #' If FALSE, only aggregates where ALL regions in that aggregate contain no data, are dropped. Defaults to TRUE
```

Comment:
@drnickisaac Pretty sure this should be one line higher

Reply:
@drnickisaac Or one line lower?

Perhaps we could modify it to `@param allowSitesMultiRegions An option that permits sites to be included in more than one region if TRUE. If FALSE then these sites are dropped. Defaults to FALSE.` 

The checks look good to me, although I'm not as familiar with sparta's code.

Reply:
I believe it should read:

```
#' @param rem_aggs_with_missing_regions An option which if TRUE will remove all aggregates which contain at least one region with no data.
#' If FALSE, only aggregates where ALL regions in that aggregate contain no data, are dropped. Defaults to TRUE
@param allowSitesMultiRegions An option that permits sites to be included in more than one region. If `FALSE` then these sites are dropped.
```